packit/cli/sync_to_upstream.py

```diff
@@ -0,0 +1,50 @@
+"""
+Update selected component from upstream in Fedora
+"""
+
+import logging
+import os
+
+import click
+
+from packit.api import PackitAPI
+from packit.cli.types import LocalProjectParameter
+from packit.cli.utils import cover_packit_exception
+from packit.config import pass_config, get_context_settings, get_local_package_config
+
+logger = logging.getLogger(__file__)
+
+
+@click.command("sync-upstream", context_settings=get_context_settings())
```

Comment:
For me, this is one of the most important lines of the PR: how do we name the command. `sync-upstream` does not reflect how people will use the command. I can imagine that this will be a common flow:
```
$ cd upstream-repo
$ packit <something>
```
It would make sense for me to name it
```
$ packit sync-from-downstream
```
But that's really long.

All your ideas are welcome.

Reply:
For now, I have changed into `sync_from_downstream`